requirements.txt

```diff
@@ -1,5 +1,6 @@
+pyqldb~=3.1.0
 argparse~=1.4.0
-amazon.ion~=0.5.0
+amazon.ion~=0.7.0
 boto3~=1.9
```

Comment:
The driver has boto3 pegged at [1.16.56
](https://github.com/awslabs/amazon-qldb-driver-python/blob/master/requirements.txt#L2)
should we do the same?

Reply:
I dont think this matters as this isn't a library, and is the endpoint application.